File: Java/Homework01/Homework01_01/Homework01_01.java
/*
 * File name : Homework01_01.java
 * Author : Kim Yeong Ho
 * Date : July 10, 2023
 * Major feature : 
 * - Scanner 클래스를 이요하여 원화를 입력받아 달러로 바꾸는 다음 예시와 같이 출력하는 프로그램을 작성하시요.
 * - 1달러는 1200원으로 가정하여 환산하길 바랍니다. 
 * - 원화를 입력하세요(단위 원) >> 3600
 * - 3600원은 $3.0 달러 입니다.
 */
package Homework01_01;

import java.util.Scanner;

public class Homework01_01 {
	public static void main(String[] args) {
		int krw;
		float usd;
		Scanner scanner = new Scanner(System.in);
		
		System.out.print("원화를 입력하세요(단위 원) : ");
		krw = scanner.nextInt();		
		usd = krw / 1200f;
		System.out.println(krw + "원은 $" + usd + "달러 입니다.");

		scanner.close();
	}
}

File: C/Homework01_01/Homework01_01.c
/*
* File name : Homework01_01.c
* Author : Kim Yeong Ho
* Date : July 21, 2023
* Major feature : 
* - 키와 몸무게를 입력 받아 키를 height에 저장하고, 몸무게를 weight에 저장하여 BMI지수를 계산 하는 프로그램을 작성하시오.
* - 단, 키는 cm단위로 입력 받은 후 m단위로 변환하시오.
* - BMI 지수는 실수 형태로 출력
* - cf) BMI지수 = 몸무게(kg) / 키(m)^2
*/
#include <stdio.h>

int main(void) {
	float height, weight, bmi;

	printf("BMI지수 계산기\n");

	printf("키를 입력하세요(cm) : ");
	scanf("%f", &height);

	printf("몸무게를 입력하세요(kg) : ");
	scanf("%f", &weight);

	bmi = weight / ((height * 0.01) * (height * 0.01));
	printf("BMI지수는 %f입니다.\n", bmi);

	return 0;
}
